### ISSUE TITLE
Tray icon disappeared in gnome

### ISSUE THREAD

**Issue by stefabat** [johannesjo/super-productivity#837]
* Version used: 6.3.3
* Operating System and version: Manjaro 64 bit
* Desktop Environment: Gnome 3.38.3 on Wayland

### Expected Behavior
If enabled in the settings, the tray icon should be visible and I can interact with it (show app, quit).

### Current Behavior
There is no more tray icon all of a sudden.

### Steps to Reproduce (for bugs)
1. Open the app
2. Enable/disable the tray icon
3. Nothing happens

### Console Output
Force dark mode for tray icon
{ enabled: null, showDevTools: false, mode: 'bottom' }

### Error Log (Desktop only)
There is no error log in the .config/superProductivity

Very strange that it used to work until yesterday, and suddenly it stopped. The only reason I could see is that I usually use an external monitor and I have enough space in the top bar (I have many tray icons), but yesterday I used it without the external monitor and perhaps there was not enough space to put the tray icon and it just decided to permanently disable it.

**Comment by johannesjo** [comment]
Hi there! Thanks for reporting this! Is this maybe related to changing your theme settings? Maybe the icon is there but just not visible because it has the same color as the background?

**Comment by stefabat** [comment]
No, that is not the case. When I started using the app two weeks ago I could not see the icon because indeed it was dark on the default dark top panel of Gnome. However, when hovering over it with the mouse I could interact with it, hence I knew it was there. I came here to check if somebody else posted an issue already and I found the PR for the --force-dark-tray-icon, which I then started to use and the icon was white and working.
Yesterday night I opened the app when not connected to the external monitor and did not pay attention to the tray icon. This morning, connected back to the external monitor, I noticed the icon was missing.
The weird part is that nothing has changed in my OS from yesterday night to today! No updates, change of theme, or whatever else..
